### DISCUSSION TITLE
How to change camera or at least disable interpolation?

### DISCUSSION THREAD

**Discussion by faxlab** [Sixze/ALS-Refactored#319] [Q&A]
Hi, where do I control various settings like camera interpolation? I tried adding new camera and setting it to active in blueprint but it didn`t change the camera. I also cant change the AlsCamera parent from blueprint. How do I solve it?
Also how do I turn off rolling, vaulting and other settings like that? Thanks

**Comment by yukioshima** [comment]
How about turning off the "Auto Activate" of the camera and adding a camera?
